### ISSUE TITLE
useGregorianParser is not defined

### ISSUE THREAD

**Issue by miladmeidanshahi** [fingerpich/jalali-moment#58]
**Describe the bug**
Uncaught ReferenceError: useGregorianParser is not defined

**Screenshots**
![Screenshot from 2019-05-19 11-07-24](https://user-images.githubusercontent.com/11172818/57978630-95146b00-7a26-11e9-8ada-d2b633d7f5a7.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu
 - Browser chrome
 - Version Version 74.0.3729.157 (Official Build) (64-bit)

Here is my code:
```
import moment from 'jalali-moment'

moment.locale(i18n.locale)
```

**Comment by miladmeidanshahi** [comment]
Link to fix this issue
https://github.com/fingerpich/jalali-moment/pull/54/commits/3baa4bf4b572aedfdbbf8fe827797e6103602928#r285367585

**Comment by fingerpich** [comment]
could you please provide a demo in StackBlitz?
please create PR If you know how to fix it.

**Comment by miladmeidanshahi** [comment]
I created PR check it, please

**Comment by fingerpich** [comment]
Its fixed and is available in [version 3.3.2](https://www.npmjs.com/package/jalali-moment)
Thanks
